jupyter/minimal/ubi9-python-3.12/Dockerfile.konflux.cpu

```diff
@@ -116,3 +112,11 @@ RUN echo "Installing softwares and packages" && \
 WORKDIR /opt/app-root/src
 
 ENTRYPOINT ["start-notebook.sh"]
+
+LABEL name="rhoai/odh-workbench-jupyter-minimal-cpu-py312-rhel9" \
```

Comment:
does not have e.g.

    authoritative-source-url

label any more. Is that intentional?